microcosm/tests/config/test_validation.py

```diff
@@ -90,6 +90,70 @@ def test_mock_value(self):
             ),
         ))
 
+    def test_int_list_value(self):
```

Comment:
Please include some failure tests. I don't understand why you return `[]` on failure -- and would prefer that you don't -- but if it's necessary, we should have a test that justifies (and documents) the behavior.

Reply:
👍 will do